aws/sra-test/integration-tests/aws-sdk-s3/tests/request_information_headers.rs

```diff
@@ -31,228 +128,267 @@ struct FixupPlugin {
+                ),
+                create_500_res("Sat, 01 Jun 2019 00:00:02 GMT")
+            )
+        ),
+        (
+            (
+                create_test_req(
+                 "3dfe4f26-c090-4887-8c14-7bac778bca07",
+                 "ttl=20190601T000013Z; attempt=4; max=4",
+                 "20190601T000003Z",
+                 "AWS4-HMAC-SHA256 Credential=ANOTREAL/20190601/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date;x-amz-security-token;x-amz-user-agent, Signature=e06b6d1cb43e4263aba325852d45fc4528645b2a5a48be20f3924d0bd0b2ffae"
+                ),
+                create_200_res("Sat, 01 Jun 2019 00:00:03 GMT")
+            )
+        ),
+    ]
+        .into_iter()
+        .map(Into::into)
+        .collect();
 
-    let path = "test-data/request-information-headers/three-retries_and-then-success.json";
-    let conn = dvr::ReplayingConnection::from_file(path).unwrap();
+    let (time_source, sleep) = instant_time_and_sleep(UNIX_EPOCH + Duration::from_secs(1559347200));
+    let sleep_impl: Arc<dyn AsyncSleep> = Arc::new(sleep.clone());
+    let conn = TestConnection::new(events, sleep_impl.clone());
     let config = aws_sdk_s3::Config::builder()
         .credentials_provider(Credentials::for_tests())
+        .http_connector(HttpConnector::Prebuilt(None))
+        .retry_config(RetryConfig::standard().with_max_attempts(4))
+        .timeout_config(timeout_config)
+        .sleep_impl(sleep_impl.clone())
         .region(Region::new("us-east-1"))
-        .http_connector(DynConnector::new(conn.clone()))
         .build();
     let client = Client::from_conf(config);
     let fixup = FixupPlugin {
         client: client.clone(),
-        timestamp: UNIX_EPOCH + Duration::from_secs(1624036048),
+        connection: conn.clone(),
+        time_source,
+        invocation_ids: vec![InvocationId::from_str(
+            "3dfe4f26-c090-4887-8c14-7bac778bca07",
+        )],
     };
 
-    let resp = dbg!(
-        client
+    let res = client
+        .list_objects_v2()
+        .bucket("test-bucket")
+        .send_orchestrator_with_plugin(Some(fixup))
+        .await
+        .expect("valid e2e test");
+
+    // Assert that we slept three times, for 1 sec each.
+    assert_eq!(
+        vec![
+            Duration::from_secs(1),
+            Duration::from_secs(1),
+            Duration::from_secs(1)
+        ],
+        sleep.logs()
+    );
+
+    assert_eq!(res.name(), Some("test-bucket"));
+    conn.assert_requests_match(&[]);
+}
+
+// # Client makes 3 separate SDK operation invocations
+// # All succeed on first attempt.
+// # Fast network, latency + server time is less than one second.
+// # No clock skew
+// # Client waits 1 second between attempts.
+// TODO(orchestratorCrossOperationState) We don't support cross-operation state yet, so we can't calculate the service time skew for operations being sent after the first one.
```

Comment:
Suggest using this TODO tag so we can continue to code search for these.
```suggestion
// TODO(enableNewSmithyRuntime) We don't support cross-operation state yet, so we can't calculate the service time skew for operations being sent after the first one.
```